modules/tinymce/src/core/test/ts/browser/caret/FakeCaretTest.ts

```diff
@@ -122,4 +122,22 @@ describe('browser.tinymce.core.caret.FakeCaretTest', () => {
     assert.isFalse(isFakeCaretTarget(createElement('<p contenteditable="false"></p>', false).dom), 'Should not have fake caret since context is noneditable');
     assert.equal(isFakeCaretTarget(createElement('<table></table>').dom), isFakeCaretTableBrowser(), 'Should on some browsers need a fake caret');
   });
+
+  it('TINY-10314: fakeCaretContainer after/before a block should have caret-color setted to `transparent` to avoid double caret in FireFox', () => {
```

Comment:
```suggestion
  it('TINY-10314: fakeCaretContainer after/before a block should have caret-color set to `transparent` to avoid double caret in FireFox', () => {
```

Reply:
changed